.gitignore

```diff
@@ -44,4 +44,4 @@ todo.txt
 coverage_report/
 test/reports/*
 yarn-error.log
-
+yarn.lock
```

Comment:
Added this here since I have manually updated a lockfile and since regenerating a new one breaks it, we should keep a good track of it.